docs/source/changelog.rst

```diff
@@ -110,6 +110,8 @@ Other additions
 - The tab icon in the browser now changes to indicate when the kernel is busy
   (:ghpull:`1837`).
 
+- Load server extensions with ConfigManager so that merge happens recursively, unlike normal config values to make it load more consistenly with frontend extensions(:ghpull:`2108`).
```

Comment:
...values, to...

missing 't' in consistently

Make sure line length is 80 characters or less.